src/uniprotkb/components/entry/SimilarProteins.tsx

```diff
@@ -0,0 +1,123 @@
+import { Loader, Message, Tabs, Tab, Card, Button } from 'franklin-sites';
+import { FC, useMemo } from 'react';
+import { groupBy } from 'lodash-es';
+import { generatePath, Link } from 'react-router-dom';
+import { getClustersForProteins } from '../../../shared/config/apiUrls';
+import useDataApi from '../../../shared/hooks/useDataApi';
+import { UniRefLiteAPIModel } from '../../../uniref/adapters/uniRefConverter';
+import EntrySection, {
+  getEntrySectionNameAndId,
+} from '../../types/entrySection';
+import { Location, LocationToPath } from '../../../app/config/urls';
+
+const SimilarProteins: FC<{
+  isoforms: { isoforms: string[] };
+  primaryAccession: string;
+}> = ({ isoforms, primaryAccession }) => {
+  const searchUrl = getClustersForProteins([
+    primaryAccession,
+    ...isoforms.isoforms,
+  ]);
+  // Get the clusters in which the canonical and isoforms are found
+  const { loading, data, error } = useDataApi<{
+    results: UniRefLiteAPIModel[];
+  }>(searchUrl);
```

Comment:
I'm guessing the default query has a result limit of 25 by default.
It would be good to have an idea of the maximum number of clusters one entry can belong to to know if we need to ask for more results.
Also, to mitigate this issue, maybe we should have 3 different calls for each tab/identity level. That would reduce network payload and also spread that 25 results limit across the 3 different identity levels

Reply:
Good point about the limit, I'll check.

I wanted to wait until part II to decide on whether to do 1 call per tab or not - I will be using the accession service to get extra information about all the members which are displayed....and I'm hoping the list of members as strings won't go away once the new model is released!

Reply:
I can't remember for sure what was decided, but I think the list of members as a string might go away, in favour of just using the `/members` endpoint, but that endpoint should give you enough information about the members themselves to not need to use the accession endpoint

Reply:
Issue then is that I would have to make a request per cluster and per isoform so not as good as one request for all clusters and 1 request for all members. I'm hoping it doesn't go away as part of the `uniref/search` service as this would be a problem for the results table too?

Reply:
Well, the current way we are using the members column of the uniref configuration is working is not correct, as it's alway ever displaying up to 10 members only, so in the search results it always appears as that there is only ever up to 10 members in an entry...